client/container_start.go

```diff
@@ -9,7 +9,9 @@ import (
 // ContainerStart sends a request to the docker daemon to start a container.
 func (cli *Client) ContainerStart(ctx context.Context, containerID string, checkpointID string) error {
 	query := url.Values{}
-	query.Set("checkpoint", checkpointID)
+	if len(checkpointID) != 0 {
```

Comment:
I'll do the vendoring later after this. I'm planning to check other APIs, to see if they have same problem. BUT, I really hope not to introduce unnecessary break to existing APIs.  :smile: